src/coreclr/jit/emitxarch.cpp

```diff
@@ -144,6 +144,97 @@ bool emitter::IsDstSrcSrcAVXInstruction(instruction ins)
     return ((CodeGenInterface::instInfo[ins] & INS_Flags_IsDstSrcSrcAVXInstruction) != 0) && IsAVXInstruction(ins);
 }
 
+//------------------------------------------------------------------------
+// IsWriteZFFlags: check if the instruction write the
+//     ZF flag.
+//
+// Arguments:
+//    ins - instruction to test
+//
+// Return Value:
+//    true if instruction writes the ZF flag, false otherwise.
+//
+bool emitter::IsWriteZFFlags(instruction ins)
```

Comment:
I wonder if just `emitter::WritesZF` or `emitter::WritesZeroFlag` would be better?

Reply:
I don't have any preference, but we will have to follow similar naming convention for other methods that checks for more than 2 flags like `IsResetCarryAndOverflowFlags`, etc.